lte/gateway/c/core/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -4762,7 +4763,9 @@ void mme_app_handle_mme_init_local_deactivation(
       ue_context_p, false, bearer_deactv_req_p->ebi,
       bearer_deactv_req_p->no_of_bearers, pdn_context->s_gw_teid_s11_s4,
       REQUEST_ACCEPTED, pdn_context->route_s11_messages_to_s8_task, true);
-  ue_context_p->nb_delete_bearer_cmd--;
+  if (ue_context_p->nb_delete_bearer_cmd) {
```

Comment:
Use explicit check for non-zero value. Further, shouldn't this check  happen before we reach this point in code?

```suggestion
  if (ue_context_p->nb_delete_bearer_cmd > 0) {
```